UnityProject/Assets/Scripts/Equipment/ObjectPool.cs

```diff
@@ -14,13 +14,16 @@ public class ObjectPool : MonoBehaviour
 
 		public void AddGameObject(GameObject obj)
 		{
-			obj.transform.position = transform.position;
-			obj.transform.parent = transform;
+//            obj.transform.position = transform.position;
+//            obj.transform.parent = transform;
+			var objTransform = obj.GetComponent<CustomNetTransform>();
+			objTransform.DisappearFromWorldServer();
+//            objTransform.SetParent(transform);

```

Comment:
Why are there so many commented out rows of code here?
Shouldn't we just get rid of those when we use GIT anyway?

Reply:
We might have to reenable ObjectPool parenting at some point, though I'm not sure of that; leaving these as a reminder